staticaddr/deposit/fsm.go

```diff
@@ -33,6 +33,14 @@ var (
 
 	Withdrawn = fsm.StateType("Withdrawn")
 
+	LoopingIn = fsm.StateType("LoopingIn")
```

Comment:
I propose to add godocs with short descriptions of what state is what to all elements of this var group.